pkg/apis/rbac/helpers.go

```diff
@@ -254,3 +254,63 @@ func (r *ClusterRoleBindingBuilder) Binding() (ClusterRoleBinding, error) {
 
 	return r.ClusterRoleBinding, nil
 }
+
+// +k8s:deepcopy-gen=false
```

Comment:
I'm not a huge fan of having helpers in the internal types api packages. @deads2k 

I guess you're not making it worse, so I'm not asking for you to solve this problem.